enf-reducer/tests/cut-unncessary-and-test.metta

```diff
@@ -0,0 +1,48 @@
+! (register-module! ../../../metta-moses-reduction) 
+! (import! &self metta-moses-reduction:enf-reducer:rte-helpers)
+! (import! &self metta-moses-reduction:utilities:general-helper-functions)
+! (import! &self metta-moses-reduction:enf-reducer:cut-unnecessary-and)
+
+
+
+    ;Test 01
+!(assertEqual 
+    (AndCut (AND (OR (AND B C) (AND E F))))
+    ((AND B C) (AND E F))
+)
+
+    ;Test 02
+!(assertEqual 
+    (AndCut (AND (OR (AND C D) (AND C F))))
+    (AND (OR (AND C D) (AND C F)))
+)
+
+    ;Test 03
+!(assertEqual
+    (AndCut (AND (OR (AND B C) (AND E F) (AND G H))))
+    ((AND B C) (AND E F) (AND G H))
+)
+
+    ;Test 04
+!(assertEqual
+    (AndCut (AND (OR (AND B C) (AND E F)) (OR (AND G H) (AND I J))))
+    (AndCut (AND (OR (AND B C) (AND E F)) (OR (AND G H) (AND I J))))
```

Comment:
Why is this test case here? Is it failing for this input?

Reply:
no it works fine, the function name was added by mistake, luckly it wasn't casing error _:)_